CosmicReco/src/CosmicTrackMCInfo.cc

```diff
@@ -1,8 +1,13 @@
 #include "CosmicReco/inc/CosmicTrackMCInfo.hh"
+
 CosmicTrackMCInfo::CosmicTrackMCInfo(){
      TrueTheta =0;
      TruePhi =0;
-	
+     
+     TruePhiSIM = 0;
+     TrueThetaSIM = 0;
+     TrueMomentum = 0;
+
```

Comment:
Two things:

1) These can be initialized in the class declaration ( in the .hh file)
        double TruePhiSIM = 0.;
    If something can be initialized that way we want everyone to do it.

2) If something cannot be initialized in the header, then use the intializer syntax ( ie the colon after the close ")" of the function arguments ).

Neither of these are big deal for this particular class but if you just do it everytime then you don't need to remember to do something special in the cases that do matter.

Reply:
ah yes sure, sorry, will do that